proximo/proximo_url.go

```diff
@@ -28,6 +29,29 @@ func newProximoSink(u *url.URL) (substrate.AsyncMessageSink, error) {
 		Topic:  topic,
 	}
 
+	if q.Get("keep-alive-time") != "" {
```

Comment:
You can extract this bit that creates keep alive config to a function and use it for both source and sink

Reply:
Can you also please add tests for these?